bootcd/genesis.ks.template

```diff
@@ -220,6 +220,8 @@ echo ">>>>> Removing bloat from /usr/share"
 rm -rf /usr/share/doc
 rm -rf /usr/share/icons
 
+sed -e 's|ACTIVE_CONSOLES.*|ACTIVE_CONSOLES="/dev/tty1 /dev/ttyS0 /dev/ttyS1"|' -i /etc/sysconfig/init
```

Comment:
Now I see https://github.com/tumblr/genesis/blob/master/bootcd/rpms/genesis_scripts/src/sysconfig-init.diff which is where it's being explicitly locked down to `tty1` per @byxorna's explanation below. I need to poke a little deeper to see if I can untangle the launching of all the automated functionality from making logging in possible.